autoreject/autoreject.py

```diff
@@ -12,8 +12,8 @@
 from mne.utils import logger, ProgressBar
 
 from sklearn.base import BaseEstimator
-from sklearn.grid_search import RandomizedSearchCV
-from sklearn.cross_validation import KFold
+from sklearn.model_selection import RandomizedSearchCV
+from sklearn.model_selection import KFold
```

Comment:
so this creates a dependency on bleeding edge `scikit-learn` version? can you make it happen without creating the dependency?